src/main/java/PeeKay.java

```diff
@@ -0,0 +1,90 @@
+import java.util.Scanner;
+
+public class PeeKay {
+    static String indent = "     ";
+    static String line = "    ____________________________________________________________";
+    static Task[] list = new Task[100];
+    static int count = 0;
+
+    public static void echo(String input) {
+        if (input.equals("bye")) {
+            System.out.println(indent + "Bye. Hope to see you again soon!");
+        } else {
+            System.out.println(indent + "Got it. I've added this task: ");
+            System.out.println((indent + "  " + list[count - 1].toString()));
+            String task = count == 1 ? "task " : "tasks ";
+            System.out.println(indent + "Now you have " + count + " " + task + "in the list.");
+        }
+    }
+
+    public static void createTask(String input) {
+        if (input.contains("todo")) {
+            list[count] = new ToDo(input.substring(input.indexOf("todo ") + 5));
+            count ++;
+            echo(input);
+        } else if (input.contains("deadline")) {
+            String description = input.replaceFirst("deadline ", "");
+            int by = description.indexOf("/");
+            String Date = description.substring(by + 4);
+            list[count] = new Deadline(description.substring(0, by - 1), Date);
+            count++;
+            echo(input);
+        } else if (input.contains("event")){
+            String description = input.replaceFirst("event ", "");
+            int from = description.indexOf("/from");
+            int by = description.indexOf("/to");
+            String startDate = description.substring(from + 6, by - 1);
```

Comment:
I think it would be helpful to add some comments throughout the code to explain certain steps for better understanding! a great code overall!!